.github/workflows/ci.yml

```diff
@@ -69,10 +69,4 @@ jobs:
             ${{ runner.os }}-
 
       - run: npm ci
-      - uses: wearerequired/lint-action@v1
-        with:
-          github_token: ${{ secrets.github_token }}
-          eslint: true
-          eslint_dir: 'src'
-          prettier: true
-          prettier_dir: 'src'
+      - run: npm run lint
```

Comment:
I replaced this lint action with the native `npm run lint` command because the latter's warning/error output is now also parsed and displayed by Github (which was the original reason for that custom Github lint-action).